cmd/polaris/fix.go

```diff
@@ -51,12 +52,27 @@ var fixCommand = &cobra.Command{
 			err := cmd.Help()
 			panic(err)
 		}
-
-		baseDir := filesPath + "/"
-		yamlFiles, err := getYamlFiles(baseDir)
+		var yamlFiles []string
+		fileInfo, err := os.Stat(filesPath)
 		if err != nil {
 			panic(err)
 		}
+		if fileInfo.IsDir() {
+			baseDir := filesPath
+			if !strings.HasSuffix(filesPath, "/") {
+				baseDir = baseDir + "/"
+			}
+			yamlFiles, err = getYamlFiles(baseDir)
+			if err != nil {
+				panic(err)
+			}
+		} else {
+			if strings.HasSuffix(filesPath, ".yaml") || strings.HasSuffix(filesPath, ".yml") {
+				yamlFiles = append(yamlFiles, filesPath)
+			} else {
+				logrus.Error("Fix command only works with yaml/yml files")
+			}
```

Comment:
I'm not normally a huge fan of expecting file extensions since they are not actually required. Can we instead just make sure it's a readable file and then try to read it? What happens if we pass a non-yaml file through?

Reply:
just removed that file extension check... There is a read file below that will make sure the file is readable.